build/teamcity/cockroach/ci/tests/acceptance.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+dir="$(dirname $(dirname $(dirname $(dirname $(dirname "${0}")))))"
+
+source "$dir/teamcity-support.sh"  # For $root
+source "$dir/teamcity-bazel-support.sh"  # For run_bazel
+
+tc_prepare
+set -x
+
+tc_start_block "Run acceptance"
```

Comment:
Is this tested? I don't see an acceptance build config in TC.

Reply:
It's "under construction" [here](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_ScratchProjectPutTcExperimentsInHere_RailBazelAcceptance?branch=68838&mode=builds) :)